bonfire/Modules/Tools/Controllers/LogsController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Bonfire\Tools\Controllers;
+
+use App\Controllers\AdminController;
+use CodeIgniter\CodeIgniter;
+use Bonfire\Modules\Tools\Libraries\Logs;
+
+class LogsController extends AdminController
+{
+    protected $theme = 'Admin';
+
+    protected $viewPrefix = 'Bonfire\Modules\Tools\Views\\';
+
+    protected $logsPath = WRITEPATH . 'logs/';
+
+    public function __construct()
+    {
+        $this->logsLimit = service('settings')->get('Logs.paginationLimit');
+
+        $this->logsHandler = new Logs();
+    }
+
+    /**
+     * Displays all logs.
+     *
+     * @return string
+     */
+    public function index()
+    {
+        // Load the Log Files.
+        $logs = get_filenames($this->logsPath);
```

Comment:
Where is the `file_system` handler loaded? I think that might have been overlooked.